example_bytecode_test.go

```diff
@@ -1,4 +1,4 @@
-package duktape_test
+package duktape
```

Comment:
@pytlesk4, in this case, we won't be able to test internals as we do it now. Also, we will have to rewrite all the tests as a separate package. Are you sure that it's good? I am not.